### ISSUE TITLE
late meeting being picked up as first meeting if new one has been created after

### ISSUE THREAD

**Issue by snakey85** [andrewwippler/rpi-wayfinding#19]
If a meeting had been booked for late in the day say 7pm and at 9am someone booked the meeting room at 11am it was always showing the meeting at 7pm as the first/next scheduled meeting instead of the 11am one.

I have changed cron_regulalry.php to combat this as below:

//check DB for rooms
		$sql_check = "SELECT * FROM
					(
						SELECT*
						FROM events
						ORDER BY start
					) AS events 
					  WHERE Room='". $r["name"] . "'
					  LIMIT 3"; // limiting to 3 requests because we do not need any more.

**Comment by andrewwippler** [comment]
Cron_daily searches for the events of the day from each room and places them into the database. If events are being scheduled after you run cron_daily, run cron_daily with the force_sync.

**Comment by snakey85** [comment]
Hi Andrew,

Yes we run cron_daily hourly it's not so much the sync but that it was taking the first meeting. So in the example we had a meeting scheduled at 7pm which was picked up the day before. Since the cron_daily was another meeting was added at 11am so we ran cron_daily again which added the new meeting in to the SQL but the program always saw the meeting at 7pm (the first one it entered in to the database) as being the first meeting.

The only way we found to sort this error was the above but i'm not 100% sure this will only ever pick up the first 3 meetings of the day. I will do some further testing.
